tests/automation-framework/rancherclient/rancherclient.go

```diff
@@ -0,0 +1,79 @@
+package rancherclient
+
+import (
+	projectClient "github.com/rancher/rancher/tests/automation-framework/client/generated/project/v3"
+	"github.com/rancher/rancher/tests/automation-framework/clientbase"
+	"github.com/rancher/rancher/tests/automation-framework/config"
+	managementClient "github.com/rancher/rancher/tests/automation-framework/management"
+	"github.com/rancher/rancher/tests/automation-framework/testsession"
+	"k8s.io/client-go/rest"
+)
+
+type RancherClient struct {
```

Comment:
We are in the rancherclient package so we can rename this to just `client`

Reply:
Yes for sure, I just misunderstood what you wanted renamed. I'll make the change.